geode-docs/managing/security/implementing_authentication.html.md.erb

```diff
@@ -23,151 +23,150 @@ Authentication lends a measure of security to a cluster
 The `authenticate` method is expected to either return an object
 representing a principal or throw an `AuthenticationFailedException`.
 
 A well-designed `authenticate` method will have a set of known user and password pairs that can be
-compared to the credential presented or will have a way of obtaining those pairs.
+compared to the credentials presented or will have a way of obtaining those pairs.
 
-## How a Server Sets Its Credential
+## <a id="authentication-server-set-creds"></a>How a Server Sets Its Credentials
 
-In order to connect with a locator that does authentication,
-a server will need to set its credential, composed of the two properties
+In order to connect with a locator that performs authentication,
+a server must set its credentials, a username and password specified as the two properties
 `security-username` and `security-password`.
-Choose one of these two ways to accomplish this:
-
-- Set `security-username` and `security-password` in the server's
-`gfsecurity.properties` file that will be read upon server start up,
-as in the example
-
-    ``` pre
-    security-username=admin
-    security-password=xyz1234
-    ```
-    The user name and password are stored in the clear, so the
-    `gfsecurity.properties` file must be protected by restricting access with
-    file system permissions.
-
-- Implement `AuthInitialize` interface for the server.
-    
-    - Set the property `security-peer-auth-init`,
-    so that an object of the class that implements the `AuthInitialize`
-    interface will be instantiated.
-    Set the property to one of these two values:
-
-        - Set property `security-peer-auth-init` to the fully-qualified
-        class name that implements the `AuthInitialize` interface
-        as in the example
-
-        ``` pre
-        security-peer-auth-init=com.example.security.ServerAuthenticate
-        ```
-        - Set property `security-peer-auth-init` to the fully-qualified
-        method name of a method that instantiates an object of the class
-        that implements the `AuthInitialize` interface
-        as in the example
-
-        ``` pre
-        security-peer-auth-init=com.example.security.ServerAuthenticate.create
-        ```
-
-    - Implement the `getCredentials` method within the `AuthInitialize`
-    interface to acquire values for
-    the `security-username` and `security-password` properties
-    in whatever way it wishes.
-    It might look up values in a database or another external resource.
-
-Gateway senders and receivers communicate as a component of their
-server member.
-Therefore, the credential of the server become those of the gateway
-sender or receiver.
-
-## How a Client Cache Sets Its Credential
-
-In order to connect with a locator or a server that does authentication,
-a client will need to set its credential, composed of the two properties
+Choose one of two ways to set the server credentials:
+
+- Add settings to the server properties file, or
```

Comment:
change to `Add settings to the server properties file if it's simple username/password combination`

Reply:
Thanks, @jinmeiliao Change incorporated.